drivers/processor/processr/cpu.inf

```diff
@@ -121,7 +121,7 @@ LoadOrderGroup = Extended Base
 
 [Strings]
 
-ReactOS = "ReactOS Foundation"
+ReactOS = "ReactOS Team"
```

Comment:
See above. A consistent "ReactOS Project" everywhere is better after all.

Reply:
We already have most translations for "ReactOS Team" in the INF files, this change would be breaking one. If we want to keep consistency so much, I think it would be better to do in a separate commit.